README.md

```diff
@@ -1,5 +1,7 @@
 # ParSeq
 
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0) [![Build Status](https://secure.travis-ci.org/linkedin/parseq.png?branch=master)](http://travis-ci.org/linkedin/parseq) [![Download](https://api.bintray.com/packages/linkedin/maven/parseq/images/download.svg)](https://bintray.com/linkedin/maven/parseq/_latestVersion)
```

Comment:
is the license in the end required after this? if not line 118-120 can be removed.